Examples/VisionExamples/Examples/ExternalCameraViewController.swift

```diff
@@ -3,7 +3,13 @@ import MapboxVision
 import UIKit
 
 /**
- * "External camera" example demonstrates how to create a custom source of video stream and pass it to `VisionManager`.
+ * "External camera" example demonstrates how to create a custom source of a video stream
+ * and pass it to `VisionManager`.
+ * **Important:** To enable most of the features of Vision SDK like world description, AR and Safety, the object
```

Comment:
when implementing through the internal camera do i need to add CameraParameters too? if yes: how?